tests/app/helpers.py

```diff
@@ -100,3 +111,9 @@ def default_service(**kwargs):
     return {
         "service": service
     }
+
+
+def assert_response_status(response, expected_status):
```

Comment:
Rather than doing this, in (all?) our other repositories we've purged all nose tests like `assert_equal()` and gone to straight-up asserts, which give perfectly intelligible error messages without needing to write our own wrapper method.

ie. just inline `assert response.status_code == 200` works fine.  I'm slightly surprised we still have a bunch of `assert_equal`s in here, I thought we'd done away with them everywhere at some point, but maybe unloved search API missed out.

For consistency we should definitely purge nose tests from this repo - not necessarily for this PR though!  But I don't see any need for this wrapper round `assert`.

Reply:
If it doesn't go in this PR let's make a tech-debt ticket, or even just do it now.

Reply:
Ah, so the reason I made a wrapper here, was because when I had a bare assert, you didn't get to see what the response (data/content) actually was.

So you have a test that fails (i.e. wrong status) but the useful error message is completely swallowed. So in this case I think a custom function serves a vaguely useful purpose?

Reply:
You can add the useful message to the assertions in-line in the tests where it seems useful (for just one test here so far), and then it's not restricted to only for assertions about  response codes.

I don't completely hate the wrapper function, but as it's only used once it feels like that could be just as well inlined.

It's not a deal-breaker on this PR at all if you like it :)

Reply:
Hmmm yeah so I intended that we would go on to use it more. How would you feel if I went and updated a few more places we assert about the status (but fail to emit any useful info about what happened)? Otherwise yeah I guess it's just a certain amount of repeated boilerplate (shrug).

Reply:
If you're sure it's going to get used more then I'm happy for it to go in as-is.